src/Plugin/Field/FieldFormatter/EDTFFormatter.php

```diff
@@ -27,7 +27,7 @@ class EDTFFormatter extends FormatterBase {
    *
    * @var array
    */
-  private $DELIMITERS = [
+  private const $DELIMITERS = [
```

Comment:
Remove the `$`. PHP constants don't use the dollar-sign signifier. It results in an error: `ParseError: syntax error, unexpected '$DELIMITERS' (T_VARIABLE)`

Reply:
:man_facepalming: